File: Tareas Java/Operaciones con Arreglos vb/Operaciones con Arreglos vb/Form1.vb
Public Class Form1
    Private Sub btnAceptar_Click(sender As Object, e As EventArgs) Handles btnAceptar.Click
        Dim formularioDeRecepcion As New FormRecepcionDatos()


        If txtCantidad.Text = "" Then
            'MsgBox()

        Else
            formularioDeRecepcion.cantidad = Convert.ToInt32(txtCantidad.Text)
            Me.Hide()
            formularioDeRecepcion.Show()
        End If


    End Sub
End Class

File: Tareas Java/Operaciones con Arreglos Csharp/Operaciones con Arreglos Csharp/Forms/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using Operaciones_con_Arreglos_Csharp.Forms;

namespace Operaciones_con_Arreglos_Csharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }

        private void btnAceptar_Click(object sender, EventArgs e)
        {
            FormRecepcionDatos formularioDeRecepcion = new FormRecepcionDatos();


            if (txtCantidad.Text == "") {
                //MsgBox()
            }
            else {
                formularioDeRecepcion.cantidad = Convert.ToInt32(txtCantidad.Text);
                this.Hide();
                formularioDeRecepcion.Show();
            }
        }
    }
}
